driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedGridFSHelper.java

```diff
@@ -117,4 +161,8 @@ public OperationResult executeUpload(final BsonDocument operation) {
             return OperationResult.of(e);
         }
     }
+
+    Document asDocument(final BsonDocument bsonDocument) {
```

Comment:
[A question not directly related to the PR changes]

We need this method because `GridFSUploadOptions.metadata` accepts `Document` instead of accepting `Bson`. Was there a reason why accepting `Bson` instead was considered a worse idea, or is this an unnecessary API limitation in retrospect?

Reply:
It seems like an unnecessary limitation.